charts/harvester-cloud-provider/harvester-cloud-provider/0.1.300+up0.1.3/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
+# negation (prefixed with !). Only one pattern per line.
+.DS_Store
+# Common VCS dirs
```

Comment:
Also, is there a `dev-v2.5` PR for this, or is it just for Rancher 2.6+? I assume it is the latter but wanted to confirm.

Reply:
It's just for the version After v2.6.1. The Harvester cloud provider can only be used in the clusters spinned up by the Harvester node driver.